### PR TITLE
new semantic of org.eolang.error object

### PR DESCRIPTION
**Pull request by Graur**
#838

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 please, have a look

File: eo-runtime/src/main/java/EOorg/EOeolang/EOerror.java [hunk 1]
```diff
@@ -114,4 +128,34 @@ public Attr attr(String name) {
...
+         */
+        private static final long serialVersionUID = 1735493012609760997L;
+        /**
+         * Sigma.
+         */
+        public final Phi sigma;
+        /**
+         * Exception.
+         */
+        public final Phi exception;
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOerror.java:147]: @Graur public attribute? :) we don't do this, ever.

File: eo-runtime/src/main/java/EOorg/EOeolang/EOerror.java [hunk 2]
```diff
@@ -114,4 +128,34 @@ public Attr attr(String name) {
         }
     }
 
+    /**
+     * When exception happens.
+     * @since 0.19
+     */
+    public static class ExError extends ExAbstract {
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOerror.java:135]: @Graur I would put it next to other `Ex*` classes, in `org.eolang` package

**Comment by yegor256** [comment]
@Graur very good! a few small comments above

**Comment by yegor256** [comment]
@Graur also, there is a section in the paper about "division by zero" problem. I think, it should also be modified.

**Comment by yegor256** [comment]
@Graur also, we have this piece of text in the paper:

Screen Shot 2022-07-10 at 9 39 50 AM

we have to modify it too

**Comment by Graur** [comment]
> @Graur also, we have this piece of text in the paper:
> 
> Screen Shot 2022-07-10 at 9 39 50 AM
> 
> we have to modify it too

@yegor256 I don't quite understand what part of this needs to be fixed. Dataization of "error" objects causes the program execution to stop Could you clarify which part of this paper is wrong?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Graur ignore me, I was wrong

**Pull request merged**